package.json

```diff
@@ -120,6 +119,7 @@
     "redux": "3.7.2",
     "redux-form": "5.3.4",
     "redux-saga": "0.9.5",
+    "regexp-quote": "^0.0.0",
```

Comment:
porque?

Reply:
Idk, this morning this was happening... 
https://github.com/punkave/sanitize-html/issues/205